naturf/tests/test_nodes.py

```diff
@@ -11,7 +11,7 @@
 
 
 from naturf.driver import Model
-from naturf.nodes import angle_in_degrees_to_neighbor
+from naturf.nodes import angle_in_degrees_to_neighbor, orientation_to_neighbor
```

Comment:
Since these imports will keep growing, you may want to simply bring in `nodes`:

```python
import naturf.nodes as nodes

# some action...
nodes.orientation_to_neighbor(x, y)
```